lua/apg/modules/ghosting.lua

```diff
@@ -283,6 +314,14 @@ APG.hookRegister( mod, "OnEntityCreated", "APG_noColOnCreate", function( ent )
 					SafeSetCollisionGroup( ent, COLLISION_GROUP_INTERACTIVE )
 				end
 				pObj:RecheckCollisionFilter()
+
+			-- Need's a whitelist to allow specific mods

```

Comment:
Is this meant to still be a comment?

Reply:
Yes, I want to make a whitelist before implementing this so if there are certain entities they want to be able to spawn without physics they have a way to do so.

Reply:
Okay, cool. Could you add a quick comment to include some more info? Maybe the date/time the comment was created, and who created it? I know we can use git blame to find out who added it and when, but it makes the comment look more temporary and allows people to more easily follow up on it if you were to vanish tomorrow, for example.

Reply:
Honestly I'm not sure about this, is this really needed? I guess it couldn't hurt, though.